.gitignore

```diff
@@ -10,6 +10,9 @@ pids
 *.pid
 *.seed
 
+# API Keys
+src/client/api/google-api-key.js
```

Comment:
can you please consolidate my config file with yours so there's only one?  you can delete the path for the config file under gitinore.